File: PageObjectJava/src/test/java/PageObjects/AutomationPracticeStore/NotLoggedInException.java
package PageObjects.AutomationPracticeStore;

import org.openqa.selenium.WebDriverException;

public class NotLoggedInException extends WebDriverException {

    public NotLoggedInException(String message){
        super(message);
    }

    public NotLoggedInException(String message, Throwable throwable){
        super(message, throwable);
    }

}

File: PageObjectDotNet/AutomationPractice/PageObjects/AutomationPracticeStore/Objects/NotLoggedInException.cs
using OpenQA.Selenium;
using System;

namespace AutomationPractice.PageObjects.AutomationPracticeStore.Objects
{
    internal class NotLoggedInException : WebDriverException
    {

        public NotLoggedInException(string message) : base(message) { }

        public NotLoggedInException(string message, Exception innerException) : base(message, innerException) { }

    }
}